shenyu-admin/src/main/java/org/apache/shenyu/admin/transfer/DashboardUserTransfer.java

```diff
@@ -88,7 +88,7 @@ public DashboardUserEditVO transfer2EditVO(final DashboardUserVO dashboardUserVO
                     vo.setDateCreated(data.getDateCreated());
                     vo.setDateUpdated(data.getDateUpdated());
                     return vo;
-                }).orElseGet(null);
+                }).orElse(null);
```

Comment:
why not modify orElseGet to orElse

Reply:
when I use `orElseGet(()->null)`, IDEA tells me to simplify to `orElse(null)`

Reply:
I think `lazy evaluation` is not needed, not create any instances here.